src/datastore/sync.js

```diff
@@ -136,8 +133,8 @@ export class Sync {
   }
 
   async push(query, options) {
-    const network = new NetworkStore(this.collectionName);
-    const cache = new DataStoreCache(this.collectionName, this.tag);
+    const network = new Network(this.collectionName);
+    const cache = new Cache(this.collectionName, this.tag);
```

Comment:
will this cause breaking changes?

Reply:
@heyzooi Yes, in the js-sdk, the move to `deprecate` the Cache store is a breaking change, largely because the `find()` API is removing its use of observables.

Reply:
No this will not. This is all internal.